### DISCUSSION TITLE
Are public vaultwarden sites legit?

### DISCUSSION THREAD

**Discussion by dcardin** [dani-garcia/vaultwarden#4564] [General]
I self host, so I don't really care, but I stumbled upon that site and it looks fishy to me. Very simple website, registrar hiding everything etc. Looks to me like a good way to set up a man in the middle type of attack. If illegitimate, is there anything that should be done?

**Comment by BlackDex** [comment]
It has been mentioned several times already. Non of the publicly hosted and accessible Vaultwarden sites have any association with the project besides the fact that they use this code.

We do not endorse any such site's, nor wil we ever run a server our self for the public. We also cannot guarantee what kind of code is running on those sites.

**Reply by dcardin** [reply]
Yeah that makes sense. I looked for previous discussions on it (specifically) and I could not find any, sorry for the duplicate question. I feel bad for those who will fall for it though. Thanks for the quick answer!

**Reply by BlackDex** [reply]
Well, it might be safe. But that is not something we know or guarantee in any sense.
